linera-views/src/scylla_db.rs

```diff
@@ -141,14 +141,22 @@ impl ScyllaDbClient {
         // Read the value of a key
         let values = (key,);
         let query = &self.read_value;
-        let result = session.query(query.clone(), values).await?;
-        if let Some(rows) = result.rows {
-            if let Some(row) = rows.into_typed::<(Vec<u8>,)>().next() {
-                let value = row?;
-                return Ok(Some(value.0));
+        let mut paging_state = None;
+        loop {
+            let result = session
+                .query_paged(query.clone(), &values, paging_state)
+                .await?;
+            if let Some(rows) = result.rows {
+                if let Some(row) = rows.into_typed::<(Vec<u8>,)>().next() {
+                    let value = row?;
+                    return Ok(Some(value.0));
+                }
             }
+            if result.paging_state.is_none() {
+                return Ok(None);
+            }
+            paging_state = result.paging_state;
         }
-        Ok(None)
     }
```

Comment:
Can we move this change and the next one (apparently motivated by the tombstone issue) to a separate PR?

Reply:
Yes, I did move to another PR. Yes, it is motivated by the tombstone, even though I do not expect it to make a difference. Something like defensive programming.